include/zephyr/dt-bindings/pwm/stm32_pwm.h

```diff
@@ -16,10 +16,14 @@
  * @{
  */
 /** PWM complementary output pin is enabled */
-#define PWM_STM32_COMPLEMENTARY	(1U << 8)
+#define STM32_PWM_COMPLEMENTARY	(1U << 8)
+/**
+ * @deprecated Use the PWM complementary `STM32_PWM_COMPLEMENTARY` flag instead.
+ */
+#define PWM_STM32_COMPLEMENTARY	(1U << 8) __DEPRECATED_MACRO
```

Comment:
You left the macro here.

Reply:
... that's like in the commit you mention
![capture](https://user-images.githubusercontent.com/51417392/213457374-6df653ba-8c0b-4f58-8cad-26852e12150b.PNG)

Reply:
removing "__DEPRECATED_MACRO" on PWM_STM32_COMPLEMENTARY